network/builder/src/builder.rs

```diff
@@ -183,6 +183,99 @@ impl NetworkBuilder {
         }
     }
 
+    pub fn create(
```

Comment:
nit create versus new, is there a common usage of create in the codebase?

Reply:
Intent:

`new` for blindly creating the struct

`create` for applying non-trivial logic and processing from the provided inputs.   Should typically invoke `new` with the processed information.

network_builder needs `create`.  The other builders could probably get away with `new`, but this leads to a consistency question.